pkg/topic/topic.go

```diff
@@ -0,0 +1,78 @@
+package topic
+
+import (
+	"errors"
+	"fmt"
+)
+
+type TopicKind uint8
+
+const (
+	TOPIC_KIND_GROUP_MESSAGES_V1 TopicKind = iota
+	TOPIC_KIND_WELCOME_MESSAGES_V1
+	TOPIC_KIND_IDENTITY_UPDATES_V1
+	TOPIC_KIND_KEY_PACKAGES_V1
```

Comment:
Can we hard code the numeric values for these, seeing as they will need to match the client's understanding too?